src/com/projectkorra/projectkorra/GeneralMethods.java

```diff
@@ -2335,19 +2338,19 @@ public static boolean locationEqualsIgnoreDirection(final Location loc1, final L
 
 	public static boolean isLightEmitting(final Material material) {
 		switch (material) {
-			case GLOWSTONE:
-			case TORCH:
-			case SEA_LANTERN:
-			case BEACON:
-			case REDSTONE_LAMP:
-			case REDSTONE_TORCH:
-			case MAGMA_BLOCK:
-			case LAVA:
-			case JACK_O_LANTERN:
-			case END_ROD:
-				return true;
-			default:
-				return false;

```

Comment:
We will need to also add the new light-emitting blocks here. That consists of Crying Obsidian, Respawn Anchor (when charged), Shroomlight, as well as the various soul fire blocks.

Reply:
I was only focusing on compatibility and firebending, but I will go back and commit these changes.